.github/workflows/pr-smoke-testing.yaml

```
@@ -36,10 +40,29 @@ jobs:
       
-      - name: Deploy
+      - name: Setup pull secret
+        if: ${{ steps.filter.outputs.notdocs == 'true' && matrix.config == 'private-images' }}
+        run: |
+          echo "${{ secrets.PULL_SECRET }}" > /tmp/pull-secret.txt
+
+      - name: Deploy the flightctl server
         if: ${{ steps.filter.outputs.notdocs == 'true' }}
+        env: ${{
+          matrix.config == 'private-images' && {
+            'IMAGE_PULL_SECRET_PATH': '/tmp/pull-secret.txt',
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 52: mapping values are not allowed in this context

(syntax-check)